chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/activityMiddleware.tsx

```diff
@@ -45,9 +44,10 @@ const handleSystemMessage = (next: any, args: any[], card: any, systemMessageSty
         return () => false;
     }
 
+    card.activity.text = renderMarkdown(card.activity.text);
```

Comment:
we need to take the same consideration as in : https://github.com/microsoft/omnichannel-chat-widget/blob/635e2559f838936a3729eaf3718818669fca1d7d/chat-widget/src/components/livechatwidget/common/initWebChatComposer.ts#L54C45-L54C99

Reply:
we have considered https://github.com/microsoft/omnichannel-chat-widget/blob/635e2559f838936a3729eaf3718818669fca1d7d/chat-widget/src/components/livechatwidget/common/initWebChatComposer.ts#L54C45-L54C99 and passed renderMarkdown function as a refrence.